projects/ui-components/lf-metadata/field-components/field-base-parts/lf-field-base/date-time-field/date-time-field.component.css

```diff
@@ -18,15 +18,65 @@
   border-radius: 24px;
 }
 
-.datetime-icon {
+/* .datetime-icon {
   font-size: 22px;
-}
+} */
 
-/* TODO: this is only a temporary fix for Angular datetime picker style issues. Remove this if there is better solution.*/
+/* TODO: this is only a temporary fix for Angular datetime picker style issues. Remove this if there is better solution. */
 ::ng-deep ngx-mat-calendar.lf-datetime-picker-overlay .mat-calendar-body-cell {
-  position: relative;
+  position: absolute;
+  height: 100%;
 }
 
 ::ng-deep ngx-mat-calendar.lf-datetime-picker-overlay .mat-calendar-arrow {
   width: 0;
 }
+
+::ng-deep .mat-datepicker-content {
+  background-color: aliceblue;
+  box-shadow: 0px 2px 4px -1px rgba(0, 0, 0, 0.2), 0px 4px 5px 0px rgba(0, 0, 0, 0.14), 0px 1px 10px 0px rgba(0, 0, 0, 0.12);
+  display: block;
+  border-radius: 4px;
+  background-color: white;
+}
+
+
+::ng-deep .mat-calendar-table-header-divider:after {
+  background: rgba(0, 0, 0, 0.12); 
+}
+
+::ng-deep .mat-calendar-table-header th {
+  font-weight: 400;
+  font-size: 11px;
+  color: rgba(0, 0, 0, 0.54);
+}
+
+::ng-deep .mat-calendar-body-label {
+  color: rgba(0, 0, 0, 0.54);
+}
+
+::ng-deep ngx-mat-calendar.lf-datetime-picker-overlay .mat-calendar-body-cell .mat-calendar-body-cell-content {
+  color: rgba(0, 0, 0, 0.87);
+  border-color: transparent;
+}
+
+::ng-deep ngx-mat-calendar.lf-datetime-picker-overlay .mat-calendar-body-cell .mat-calendar-body-cell-content.mat-calendar-body-today {
+    border-color: var(--mat-datepicker-calendar-date-today-outline-color);
+}
+
+::ng-deep ngx-mat-calendar.lf-datetime-picker-overlay .mat-calendar-body-cell .mat-calendar-body-cell-content.mat-calendar-body-selected {
+  background-color: #3f51b5;
```

Comment:
where did you get this color from?

Reply:
This is the color used for the background when a date is selected in datepicker/date-timepicker
It is an angular colour, but instead of hardcoding it, i will set it to the angular var that the datepicker uses as well, that being 
var(--mat-datepicker-calendar-date-selected-state-background-color)